fairscale/optim/adascale.py

```diff
@@ -72,9 +72,9 @@ class AdaScale(Optimizer):
         scheduler = LambdaLR(optim, lr_lambda=...)
 
         last_epoch = 0
-        done = False
+        done = True
```

Comment:
```suggestion
        done = False
```

Reply:
Yes, this improves it. I also checked the tutorial and that is how the loop is written there as wel.